src/bin/travis/afterSuccess.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "BUILD_TYPE='$BUILD_TYPE'"
+echo "TRAVIS_BRANCH='$TRAVIS_BRANCH'"
+echo "TRAVIS_PULL_REQUEST='$TRAVIS_PULL_REQUEST'"
+
+if [ "$BUILD_TYPE" == "sbt" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+
+    if [ "$TRAVIS_BRANCH" == "develop" ]; then
+        sbt 'set test in Test := {}' publish
+
+    elif [[ "$TRAVIS_BRANCH" =~ ^[0-9\.]+_hotfix$ ]]; then
+        docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
```

Comment:
Still awaiting these credentials to be added [to Travis](https://travis-ci.org/broadinstitute/cromwell/settings).